modules/installation/proc-add-custom-app-config-file-ocp-operator.adoc

```diff
@@ -0,0 +1,95 @@
+[id="proc-add-custom-app-config-file-ocp_{context}"]
+= Adding a custom application configuration file to {ocp-short} using the Operator
+
+A custom application configuration file is a ConfigMap that you can use to change the configuration of your {product} instance to meet the needs and preferences of your organization. If you are deploying your {product-short} instance on {ocp-brand-name}, you can use the {product} Operator to add a custom application configuration file to your {ocp-short} instance by creating the ConfigMap and referencing it in the {product-short} Custom Resource.
+
+The custom application configuration file contains a sensitive environment variable, named `BACKEND_SECRET`, which is a mandatory backend authentication key that {product-short} uses to reference an environment variable defined in an {ocp-short} Secret. You must create a custom Secret, named 'secrets-rhdh', and reference it in the {product-short} Custom Resource.
+
+[NOTE]
+====
+You are responsible for protecting your {product} installation from external and unauthorized access. Manage the backend authentication key like any other secret. Meet strong password requirements, do not expose it in any configuration files, and only inject it into configuration files as an environment variable.
+====
+
+.Prerequisites
+* You have an active {ocp-brand-name} account.
+* Your administrator has installed the {product} Operator in {ocp-short}. For more information, see link:{LinkAdminGuide}[Installing the {product} Operator].
+* You have created the {product} Custom Resource in {ocp-short}.
+
+.Procedure
+. From the *Developer* perspective, select the *ConfigMaps* tab.
```

Comment:
```suggestion
. From the *Developer* perspective in the {ocp-short} web console, select the *ConfigMaps* tab.
```
Should this be "view" instead of "tab", similar to Topology?